generator/camera_node_processor.hpp

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include "generator/osm_element.hpp"
+
+#include "routing/base/followed_polyline.hpp"
+#include "routing/routing_helpers.hpp"
+
+#include "coding/file_writer.hpp"
+
+#include "base/control_flow.hpp"
+#include "base/logging.hpp"
+#include "base/string_utils.hpp"
+
+#include <cstdint>
+#include <string>
+#include <vector>
+
+namespace generator
+{
+class CameraNodeProcessor
+{
+public:
+  explicit CameraNodeProcessor(std::string const & filePath) : m_fileWriter(filePath)
+  {
+    LOG(LINFO, ("Saving information about cameras and ways, where they lie on, to:", filePath));
```

Comment:
Saving on which ways the cameras lie to

Saving the cameras-ways correspondence to

Reply:
done